Lykke.Bil2.Bcn.TransactionsExecutor/src/Lykke.Bil2.Bcn.TransactionsExecutor/Services/TransactionExecutor.cs

```diff
@@ -0,0 +1,72 @@
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+using Lykke.Bil2.Contract.Common;
+using Lykke.Bil2.Contract.TransactionsExecutor;
+using Lykke.Bil2.Contract.TransactionsExecutor.Requests;
+using Lykke.Bil2.Contract.TransactionsExecutor.Responses;
+using Lykke.Bil2.Sdk;
+using Lykke.Bil2.Sdk.Exceptions;
+using Lykke.Bil2.Sdk.TransactionsExecutor;
+using Lykke.Bil2.Sdk.TransactionsExecutor.Services;
+using Newtonsoft.Json;
+
+namespace Lykke.Bil2.Bcn.TransactionsExecutor.Services
+{
+    public class TransactionExecutor : ITransactionExecutor
+    {
+        public TransactionExecutor(/* TODO: Provide specific settings, if necessary */)
+        {
+        }
+
+        public async Task BroadcastAsync(BroadcastTransactionRequest request)
+        {
+            // TODO: broadcast transaction, throw TransactionBroadcastingException on error
+            //
+            // IMPORTANT: This method MUST NOT throw if transaction affects blockchain in any manner!
```

Comment:
Actually, any exception can be thrown, but if it not RequestValidationException or TransactionBroadcastingException, request will be not retried automatically. Let's describe all expected exceptions listed in the method description in ITransactionExecutor.

Reply:
Added exceptions description